packages/api/src/email-campaign/email-campaign.module.ts

```diff
@@ -24,7 +24,7 @@ interface EmailCampaignModuleConfig {
 @Module({})
 export class EmailCampaignModule {
     static register({ Scope, EmailCampaignContentBlock, TargetGroup, BrevoConfig }: EmailCampaignModuleConfig): DynamicModule {
-        const EmailCampaign = EmailCampaignEntityFactory.create({
+        const EmailCampaign = createEmailCampaignEntity({
```

Comment:
This should be passed from the module's config

Reply:
470e1724c6a6dd5262ddd1d5df1e61918dfe510e